src/shared/lib/localization/en.json

```diff
@@ -6,7 +6,7 @@
   "viewAll": "view all",
   "token": "Token",
   "accounts": "Accounts",
-  "blocks": "Blocks",
+  "blocks": "Blocks | Block",
```

Comment:
If you use pluralisation (` | `) inside of a message, then the key of the message (`blocks`) here should be in a singular form, i.e. `block` & `Block | Blocks`. Singular comes first here.

Reply:
Acknowledged and committed 👍🏼